packages/ember-routing-htmlbars/lib/keywords/link-to.js

```diff
@@ -315,19 +313,13 @@ export default {
   },
 
   render(morph, env, scope, params, hash, template, inverse, visitor) {
-    var attrs = merge({}, hash);
-
-    // TODO: Rewrite link-to to use arbitrary length positional params.
-    attrs.params = readArray(params);
-
-    // Used for deprecations (to tell the user what view the deprecated syntax
-    // was used in).
-    attrs.view = env.view;
-
-    // TODO: Remove once `hasBlock` is working again
-    attrs.hasBlock = !!template;
+    Ember.runInDebug(() => {
```

Comment:
Is this kosher? This is only used for an error message inside an `Ember.Deprecate` call.